### ISSUE TITLE
Typescript typings are not accurate

### ISSUE THREAD

**Issue by MacroChip** [arantes555/electron-fetch#6]
The namespace fetch is exported instead of the function fetch

**Comment by scottrippey** [comment]
I am also having this problem ... I get the following error:
```
import fetch from 'electron-fetch';
// TS Error:
// TS1192: Module '"./node_modules/electron-fetch/lib/index"' has no default export.
```
I think `export = fetch;` needs to be changed to `export default fetch;`
I will create a PR!

**Comment by Anrock** [comment]
Given the current state of things - is there a way to properly import fetch at all? Everything i've tried gives me "fetch is not a function".

**Comment by arantes555** [comment]
Should be fixed in 1.3.0
